src/wasm/wasm.cpp

```diff
@@ -376,8 +376,6 @@ void AtomicNotify::finalize() {
   }
 }
 
-void AtomicFence::finalize() { type = Type::none; }
-
```

Comment:
Why remove this, and why not add the new one?

Reply:
`Nop` doesn't have this, for example, and since both AtomicFence and Pause are glorified nops, it seemed simpler to remove this `finalize` than to add a new one.